public/app/plugins/datasource/cloudwatch/query-runner/CloudWatchLogsQueryRunner.ts

```diff
@@ -180,7 +180,7 @@ export class CloudWatchLogsQueryRunner extends CloudWatchRequest {
       refId: query?.refId || 'A', // dummy
       limit,
       startFromHead: direction !== LogRowContextQueryDirection.Backward,
-      region: query?.region || '',
+      region: this.templateSrv.replace(this.getActualRegion(query?.region)),
```

Comment:
Will this fallback to empty string if it doesn't interpolate correctly / region doesn't exist on the query object?

Reply:
Yup! `getActualRegion` handles the undefined case